pkg/openapi/manager_test.go

```diff
@@ -46,15 +72,15 @@ func Test_ValidateMutationPolicy(t *testing.T) {
 	for i, tc := range tcs {
 		policy := v1.ClusterPolicy{}
 		_ = json.Unmarshal(tc.policy, &policy)
-
 		var errMessage string
 		err := o.ValidatePolicyMutation(&policy)
 		if err != nil {
 			errMessage = err.Error()
 		}
-
-		if errMessage != tc.errMessage {
-			t.Errorf("\nTestcase [%v] failed:\nExpected Error:  %v\nGot Error:  %v", i+1, tc.errMessage, errMessage)
+		if tc.mustSucceed {
+			assert.NilError(t, err, "\nTestcase [%v] failed: Expected No Error, Got Error:  %v", i+1, errMessage)
+		} else {
+			assert.Assert(t, err != nil, "\nTestcase [%v] failed: Expected Error To have Occured.", i+1)
```

Comment:
change `Testcase [%v] failed: Expected Error To have Occured` to `Testcase [%v] failed: Expected error to have occurred`